bin/GameIndex.dbf

```diff
@@ -68,6 +68,7 @@
 -- DMABusyHack 		= 1 // Mana Khemia, Metal Saga. Denies writes to the DMAC when it's busy.
 -- VIFFIFOHack 		= 1 // Transformers Armada, Test Drive Unlimited. Fixes slow booting issue.
 -- VIF1StallHack 	= 1 // SOCOM II HUD and Spy Hunter loading hang.
+-- FMVinSoftwareHack= 1 // Silent Hill 2-3. Fixes FMVs that are obscured when using hardware rendering by switching to Software rendering while an FMV plays.
```

Comment:
vertical alignment would be nice here.

Also ``Software`` **->** ``software``.